README.md

```diff
@@ -1,12 +1,44 @@
-# Git Repository Template
+# Azure Pipelines
 
-Project repository template to set up all public projects at [Nimble](https://nimblehq.co/)
+Our common Azure Pipeline workflows used in our projects at [Nimble](https://nimblehq.co/).
+- `publish_wiki.yml`: inspired by [GitHub Actions's publish_wiki.yml](https://github.com/nimblehq/github-actions-workflows/blob/develop/.github/workflows/publish_wiki.yml) workflow to publish the wiki content to the [same GitHub's Wiki repository](https://github.com/nimblehq/azure-pipelines/blob/main/Pipelines/publish_wiki.yml#L34) of the current attached GitHub's repository in the Azure DevOps.
 
 ## Usage
 
-Clone the repository
+### Publish Wiki (GitHub)
 
-`git clone git@github.com:nimblehq/git-template.git`
+Prepare the wiki content in folder `.github/wiki/`.
+
+Create a new Azure Pipelines yml file with these configurations as below:
+- `group: 'Android Pipelines'`: the variables group to provide variables access in following steps.
+- `ref: main` (optional, default is `main`): the branch which contains the version of the publish wiki workflow in this repository.
+- `USER_NAME`: the GitHub's user name used to publish the wiki content and has access to the destination's Wiki repository.
+- `USER_EMAIL`: the GitHub's user email used to publish the wiki content and has access to the destination's Wiki repository.
+- `USER_TOKEN`: the GitHub's [Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) of the above username used to publish the wiki content with [basic `repo` scopes](https://docs.github.com/en/developers/apps/building-oauth-apps/scopes-for-oauth-apps#available-scopes).
+
+```
+trigger:
+  - develop
+pr: none
+
+variables:
+  - group: 'Android Pipelines'
+
+resources:
+  repositories:
+    - repository: publish_wiki_template
+      type: github
+      name: nimblehq/azure-pipelines
+      endpoint: nimblehq
+      ref: main
```

Comment:
We don't need `ref: main` if main is the default. Without `ref`, it will pull the `default` branch.

Reply:
@blyscuit I think explained here already. It's good to have that option in case we need to change later.

<img width="725" alt="image" src="https://user-images.githubusercontent.com/16315358/218646863-a37293ad-6561-48dc-ad66-0738b993367f.png">